java/fury-core/src/main/java/io/fury/serializer/StringSerializer.java

```diff
@@ -705,4 +705,110 @@ public String readUTF8String(MemoryBuffer buffer) {
       return new String(tmpArray, 0, numBytes, StandardCharsets.UTF_8);
     }
   }
+
+  public void writeJavaStringBuilder(MemoryBuffer buffer, StringBuilder value) {
```

Comment:
Seems unnecessary, we can provide `public void writeJDK8Ascii(MemoryBuffer buffer, char[] chars, int strLen)` and `public void writeJDK8UTF16(MemoryBuffer buffer, char[] chars, int strLen)` in StringSerializer and cann those function in this class

Reply:
Yes, it is unnecessary